### PR TITLE
remove unused old nodes (pre 0.1)

### PR DESCRIPTION
**Pull request by lumjjb**
# Description of the PR
Remove old nodes that are unused.

# PR Checklist

- [x] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
- [x] All new changes are covered by tests
- [x] If GraphQL schema is changed, `make generate` has been run
- [x] If `collectsub` protobuf has been changed, `make proto` has been run
- [x] All CI checks are passing (tests and formatting)
- [x] All dependent PRs have already been merged

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]

**Review by pxp928 — APPROVED** [review]

File: pkg/assembler/nodes.go [hunk 1]
```diff
@@ -1,515 +0,0 @@
-//
```
Comment (lumjjb) [inline pkg/assembler/nodes.go:1]: adding comment to block , made additional significant changes

**Review by mihaimaruseac — APPROVED** [review]
Looks good

File: pkg/assembler/backends/neo4j/src.go [hunk 2]
```diff
@@ -130,10 +129,6 @@ func (e *srcToType) Type() string {
 	return "SrcHasType"
 }
 
-func (e *srcToType) Nodes() (v, u assembler.GuacNode) {
-	return e.src, e.srcType
-}
-
 func (e *srcToType) Properties() map[string]interface{} {
```
[the remark this answers is not in the document]
Comment (pxp928) [reply pkg/assembler/backends/neo4j/src.go:132]: actually you can remove everything from: https://github.com/guacsec/guac/blob/e2e8ea7055c4d79102dbb3e94e71e5504d39f32e/pkg/assembler/backends/neo4j/src.go#L27-L184

File: pkg/assembler/backends/neo4j/pkg.go [hunk 3]
```diff
@@ -204,10 +195,6 @@ func (e *namespaceToName) Type() string {
 	return "PkgHasName"
 }
 
-func (e *namespaceToName) Nodes() (v, u assembler.GuacNode) {
```
Comment (pxp928) [inline pkg/assembler/backends/neo4j/pkg.go:207]: same for all these in the neo4j backend

**Review by pxp928 — COMMENTED** [review]
Couple more things cam be deleted else looks good.

File: pkg/assembler/backends/neo4j/pkg.go (hunk 3 above)
Comment (lumjjb) [reply pkg/assembler/backends/neo4j/pkg.go:207]: Removed a bunch of stuff, PTAL

File: pkg/assembler/backends/neo4j/src.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (lumjjb) [reply pkg/assembler/backends/neo4j/src.go]: removed

File: pkg/assembler/backends/neo4j/pkg.go (hunk 3 above)
Comment (lumjjb) [reply pkg/assembler/backends/neo4j/pkg.go:207]: https://github.com/guacsec/guac/pull/1035/commits/181ff6a0cd4f7250a07dc8319b61f7e843983993

File: pkg/assembler/backends/neo4j/artifact.go [hunk 4]
```diff
@@ -23,33 +23,6 @@ import (
 	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
 )
 
-// ArtifactNode is a node that represents an artifact
```
Comment (pxp928) [inline pkg/assembler/backends/neo4j/artifact.go:26]: missing deleting builder: https://github.com/guacsec/guac/blob/e2e8ea7055c4d79102dbb3e94e71e5504d39f32e/pkg/assembler/backends/neo4j/builder.go#L25-L47

Comment (lumjjb) [reply pkg/assembler/backends/neo4j/artifact.go:26]: done!

**Pull request merged**